composables/useFilteredVersions.ts

```diff
@@ -0,0 +1,43 @@
+import { Ref } from 'vue'
+
+interface Version {
+  game_versions: any[]
+  loaders: any[]
+  version_type: any
+}
+
+export default (unfilteredVersions: Ref<Version[]> | Version[], onQueryChange: () => void) => {
```

Comment:
Why not named function?

```suggestion
export default function useFilteredVersions(
  unfilteredVersions: Ref<Version[]> | Version[],
  onQueryChange: () => void,
) {
```

Reply:
No reason really, I just usually write anonymous functions with `export default`

Reply:
Does eslint not enforce consistency on this?

Reply:
It seems it doesn't, but it does remove the last comma